apps/ejabberd/src/mod_pubsub.erl

```diff
@@ -2840,6 +2885,7 @@ get_subscriptions(Host, Node, JID, Plugins) when is_list(Plugins) ->
     case Result of
         {ok, Subs} ->
             Entities = lists:flatmap(fun
+                                         %% 2-element tuples are not used by any node type probably
```

Comment:
Theoretically someone may be using custom `node_` plugin that returns such tuples. I wasn't sure whether to remove it or not. :/